server/datastore/mysql/migrations/tables/20240905140514_AddURLToSoftwareInstallers.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240905140514, Down_20240905140514)
+}
+
+func Up_20240905140514(tx *sql.Tx) error {
+	// The new 'url' column will only be set for software uploaded in batch via GitOps.
+	if _, err := tx.Exec(`
+		ALTER TABLE software_installers
+		ADD COLUMN url VARCHAR(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '';
```

Comment:
We should validate the URL length in gitops and print a meaningful message. URLs can be long.